docker-compose.yml

```diff
@@ -73,6 +79,13 @@ services:
       - tno-elastic-data:/usr/share/elasticsearch/data
     networks:
       - tno
+    # healthcheck:
+    #   test: ["CMD", "curl", "-f", "-u '$$ELASTIC_USERNAME:$$ELASTIC_PASSWORD'", "http://localhost:9200/_cat/health"]
+    #   # test: curl -f -u '${ELASTIC_USERNAME}:${ELASTIC_PASSWORD}' http://localhost:9200/_cat/health
```

Comment:
Can't get this to work presently as the username and password are not be picked up correctly.